src/sentry/static/sentry/app/components/discover/transactionsList.tsx

```diff
@@ -39,12 +42,21 @@ export type DropdownOption = {
    * The label to display in the dropdown
    */
   label: string;
+  /**
+   * Included if the option is for a trend
+   */
+  trendType?: string;
```

Comment:
Change the type here to a const of just the possible trend types.

i.e.
```typescript
trendType?: 'regression' | 'improved';
```

Though nit here being if you can use nouns for both like `'regression' | 'improvement'`

Reply:
fixed in https://github.com/getsentry/sentry/pull/22012/commits/faf0c2beffd6248668e910c5d80ee976c34bf689

Reply:
Ahh even better, there is already a type for it 👍